stdlib/test/sys/test_aarch64_target.mojo

```diff
@@ -14,24 +14,23 @@
 # COM: TODO (17471): Not all aarch64 have neon, so we need to guard against that,
 # for now just require apple-silicon.
 # REQUIRES: apple-silicon
-# RUN: %mojo -debug-level %s | FileCheck %s
+# RUN: %mojo -debug-level %s
 
 from sys.info import alignof, has_avx512f, has_neon, simdbitwidth
+from testing import assert_false, assert_true, assert_equal
 
 
 # CHECK-LABEL: test_arch_query
-fn test_arch_query():
-    print("== test_arch_query")
-
+def test_arch_query():
     # CHECK: True
-    print(has_neon())
+    assert_true(has_neon())
```

Comment:
You can remove the comments `# CHECK...`

Reply:
Thanks! Was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) I'm closing it

Reply:
Thanks! I was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) 

I think I should close it...

Reply:
Thanks! I was eyeing this ticket before realizing you have worked on most of it. I don't think it's fair for me to take this one :) 

I think I should close it and let you take the full credit...

Reply:
There is no need for you to close this pull request. Credit isn't important. You can remove the few comments that are in the code and this pull request should be good to merge

Reply:
Ah, that's so nice of you. Removed the comments and opened the pull request again.